tests/unit/components/pages/donation_form/single_page_form/PaymentSection.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { createStore } from 'vuex';
```

Comment:
Instead of the vuex store you shold import the production one:

`import { createStore } from '@src/store/donation_store';`